drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -70,6 +70,7 @@ static struct nrf5_802154_data nrf5_data;
 #define DRX_SLOT_PH 0 /* Placeholder delayed reception window ID */
 #define DRX_SLOT_RX 1 /* Actual delayed reception window ID */
 #define PH_DURATION 10 /* Duration of the placeholder window, in microseconds  */
+#define DRX_ADJUST  700 /* Correction for the receive slot start */
```

Comment:
Why this adjustment is needed? Is there some offset between timer domains used by the radio driver and Zephyr?

Reply:
I would also like to know where this value came from. Is it correct across all ranges of delayed RX duration and start times?

Reply:
The origin of the adjustment is still unknown, it was derived from trial and error testing with usual CSL configurations.

Reply:
I would clarify in the comment that it is a temporary workaround for a problem being investigated.